File: app/src/main/java/com/example/anton/sprintcalendar/PresenceType.java
package com.example.anton.sprintcalendar;

public enum PresenceType {
    NONE(0), HALF_DAY(2.5f), FULL_DAY(5);

    private float presencePercentage;

    PresenceType(float presencePercentage) {
        this.presencePercentage = presencePercentage;
    }

    public float hours() {
        return presencePercentage;
    }

    public static PresenceType safePresence(PresenceType absence) {
        return absence == null ? FULL_DAY : absence;
    }
}

File: app/src/main/kotlin/net/sourcefusion/agiletools/sprintcalendar/PresenceType.kt
package net.sourcefusion.agiletools.sprintcalendar

enum class PresenceType(val hours: Float) {
    ABSENT(0f), HALF_DAY(2.5f), HALF_DAY_MORNING(2.5f), FULL_DAY(5f);

    companion object {
        fun safePresence(absence: PresenceType?): PresenceType {
            return absence ?: FULL_DAY
        }
    }
}
